app.js

```diff
@@ -0,0 +1,35 @@
+const body = document.querySelector("body");
+const firstPage = document.querySelector(".first-page");
+const introText = document.querySelector(".intro-text");
+const eventTitles = document.querySelector(".event-titles");
+// const eventKeyResult = document.querySelector(".key-result");
+// const eventCodeResult = document.querySelector(".code-result");
+// const eventWhichResult = document.querySelector(".which-result");
+
+document.addEventListener("keydown", (event) => {
+    console.log(event);
+    firstPage.style.display = "none";
+    
+    let eventKeyResult = event.key;
+    console.log(eventKeyResult);
```

Comment:
Do you still need these console.logs in place? Or is this leftover test code?

Reply:
will comment these out